dotcom-rendering/src/server/lib/logging.ts

```diff
@@ -22,6 +27,7 @@ const logFields = (logEvent: LoggingEvent): unknown => {
 		'@version': 1,
 		level: logEvent.level.levelStr,
 		level_value: logEvent.level.level,
+		request,
```

Comment:
Nice to have request details such as Page ID on log entries even if they're not generated by the request logger.